src/api/getArticleMediaInfo.js

```diff
@@ -0,0 +1,41 @@
+import { cachedFetch, buildMwApiUrl, buildCommonsApiUrl } from 'utils'
+
+export const getArticleMediaInfo = (lang, title, fromCommon) => {
+  const params = {
+    action: 'query',
+    prop: 'imageinfo',
+    iiextmetadatafilter: 'License|LicenseShortName|ImageDescription|Artist',
+    iiextmetadatalanguage: lang,
+    iiextmetadatamultilang: 1,
+    iiprop: 'url|extmetadata',
+    titles: title
+  }
+
+  const url = fromCommon ? buildCommonsApiUrl(params) : buildMwApiUrl(lang, params)
+  return cachedFetch(url, data => {
+    const pages = data.query.pages
+    const imageInfo = pages[0].imageinfo
+
+    if (!imageInfo) {
+      return {}
+    }
+
+    const { Artist, ImageDescription, LicenseShortName } = imageInfo[0].extmetadata
+    const author = Artist && strip(Artist.value)
+    const description = ImageDescription && strip(
+      (typeof ImageDescription.value === 'string' && ImageDescription.value) ||
+      (ImageDescription.value[lang] || ImageDescription.value[Object.keys(ImageDescription.value)[0]])
+    )
+    return {
+      author,
+      description,
+      license: LicenseShortName && LicenseShortName.value,
+      filePage: imageInfo[0].descriptionshorturl
+    }
+  })
+}
+
+function strip (html) {
```

Comment:
Could you make this `const strip = html => {...}` for consistency

Reply:
I hate to be this picky... the next line has `var` but we prefer `const` or `let`

Reply:
that's alright, i didn't pay too much attention when I copy this function from others, i should double check again, thanks !